lib/components/user/new-account-wizard.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react'
 
+import FormattedAccountText from '../util/formatted-account-text'
```

Comment:
File is missing, please add.

Reply:
Sorry, the way that component looks, it is better to set the `title` prop directly on each pane below and not have that extra component.